docs/build/micro-rdk/board/_index.md

```diff
@@ -46,7 +46,7 @@ See [PWM signals on `esp32` pins](/build/micro-rdk/board/esp32/#pwm-signals-on-e
 
 For `Analog`s:
 
-- [`Read()`](/components/board/#read)
+- [`ReadAnalogReader()`](/components/board/#readanalogreader)
```

Comment:
This should stay Read, becuase Write will be moved here as well. 
https://pkg.go.dev/go.viam.com/rdk@v0.26.0/components/board#Analog